### PR TITLE
CCTRI-1892: implemented conf token support

### PR DESCRIPTION
**Pull request by MaxMhonc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,74 +127,22 @@ zappa undeploy dev
...
-on.
-
-- Set the `SECRET_KEY` environment variable for your Lambda using the
-corresponding value from the previous step.
+Now, the only thing left to do is to follow one of these URLs to navigate 
+to SecureX Threat Response page in your region and create the Pulsedive
+module using your Lambda's URL and Pulsedive API key:
+- US: https://securex.us.security.cisco.com/integrations/available/44606d8d-fc66-4ac7-815b-4448e096180f/new
+- EU: https://securex.eu.security.cisco.com/integrations/available/33b8dd88-d126-49f8-afca-3a43dee7896d/new
+- APJC: https://securex.apjc.security.cisco.com/integrations/available/213bd6b5-fd73-44db-9157-3b5156b01530/new
```
Comment (AlinaBlahun) [inline README.md:137]: Use Abuse module type ids here (now links lead to Pulsdive) https://github.com/threatgrid/integrations/tree/master/Serverless/Resources

File: README.md [hunk 2]
```diff
@@ -266,22 +214,6 @@ header set to `Bearer <JWT>`.
 
 ### JWT Payload Structure
```
Comment (AlinaBlahun) [inline README.md:215]: Should be removed as well

File: README.md [hunk 3]
```diff
@@ -266,22 +214,6 @@ header set to `Bearer <JWT>`.
...
-
-- `CTR_ENTITIES_LIMIT`
-  - Restricts the maximum number of CTIM entities of each type returned in a
-  single response per each requested observable.
-  - Applies to the following CTIM entities:
-    - `Judgement`,
-    - `Sighting`.
-  - Must be a positive integer. Defaults to `100` (if unset or incorrect).
-
 ## CTIM Mapping Specifics
```
Comment (AlinaBlahun) [inline README.md:217]: `### CTIM Mapping Specifics`

File: api/enrich.py [hunk 4]
```diff
@@ -353,10 +353,10 @@ def observe_observables():
     time_now = datetime.utcnow()
 
     # get dict with actual abuseipdb categories with titles and descriptions
```
Comment (AlinaBlahun) [inline api/enrich.py:355]: Move the comment to the code it describes

File: api/utils.py [hunk 5]
```diff
@@ -68,6 +55,67 @@ def get_auth_token():
         raise AuthorizationError(expected_errors[error.__class__])
 
 
+def get_pub_key(jwks_host, token):
```
Comment (AlinaBlahun) [inline api/utils.py:58]: `def get_public_key(jwks_host, token):`

**Review by AlinaBlahun — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MaxMhonc) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (MaxMhonc) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (MaxMhonc) [reply README.md]: Done

File: api/enrich.py (hunk 4 above)
Comment (MaxMhonc) [reply api/enrich.py]: Done

File: api/utils.py (hunk 5 above)
Comment (MaxMhonc) [reply api/utils.py]: Done

**Review by GeorgiyII — APPROVED** [review]

**Pull request merged**